projects/Mallard/src/helpers/__tests__/files.spec.ts

```diff
@@ -18,21 +17,71 @@ describe('helpers/files', () => {
     })
 
     describe('issuesToDelete', () => {
-        MockDate.set('2019-08-21')
-        it('should return items outside of the 7 days that dont follow the issue naming, or the issue index', async () => {
+        beforeEach(() => {
+            jest.resetModules()
+        })
+
+        it('should return items outside of the 7 latest issues', async () => {
+            jest.mock('src/helpers/settings', () => ({
+                getSetting: () => 7,
+            }))
+            const { issuesToDelete } = await require('../../helpers/files')
```

Comment:
Confused by this, cant this be imported at the beginning?

Reply:
I couldn't make it work by importing at the beginning.

I needed to mock a function that will return different values in different tests. The only way it seems to work by using 'require' locally.

Reply:
Can we pair on this one?